source/documentation/administration_guide/topics/The_oVirt_Engine_Rename_Tool.adoc

```diff
@@ -22,6 +22,26 @@ The `ovirt-engine-rename` command updates records of the fully qualified domain
 
 * /etc/pki/ovirt-engine/keys/apache.p12
 
+[NOTE]
+====
+Are you sure you need to do this?
+
+Since version 4.0.4, it is possible to add more names to access the engine web interface.
+
+. Make sure the names you choose can resolve to an IP address of the engine machine by adding relevant records to the DNS names or to `/etc/hosts`.
+. Run the following:
+
+  ----
+  # echo 'SSO_ALTERNATE_ENGINE_FQDNS="alias1.example.com alias2.example.com"' \
+  > /etc/ovirt-engine/engine.conf.d/99-custom-sso-setup.conf
+  # systemctl restart ovirt-engine.service
+  ----
+. List the alternate names separated by spaces.
+
+It is also possible to add IP addresses of engine host. However, using IP addresses instead of DNS names is not good practice.
```

Comment:
Perhaps change "of engine host" to "of the engine machine"? Usually "host" in oVirt refers to a hypervisor.